packages/x-codemod/src/v6.0.0/column-menu-components-rename/column-menu-components-rename.test.ts

```diff
@@ -0,0 +1,44 @@
+import path from 'path';
+import { expect } from 'chai';
+import jscodeshift from 'jscodeshift';
+import transform from '.';
+import readFile from '../../util/readFile';
+
+function read(fileName) {
+  return readFile(path.join(__dirname, fileName));
+}
+
+const TEST_FILES = ['community-root-imports', 'pro-root-imports', 'premium-root-imports'];
+
+describe('v6.0.0', () => {
+  describe('column-menu-components-rename', () => {
+    TEST_FILES.forEach((testFile) => {
+      const actualPath = `./actual-${testFile}.spec.tsx`;
+      const expectedPath = `./expected-${testFile}.spec.tsx`;
+
+      describe(`Package (${testFile.replace(/-/g, ' ')})`, () => {
```

Comment:
Nitpick, maybe you wanted to write:
```suggestion
      describe(`Package (${testFile.split(/-/g)[0]})`, () => {
```
producing `community` instead of `community root imports`? 🤔